sites/partners/src/listings/PaperListingForm/sections/Units.tsx

```diff
@@ -48,8 +48,9 @@ const FormUnits = ({ units, setUnits, disableUnitsAccordion }: UnitProps) => {
   }
 
   useEffect(() => {
-    setValue("disableUnitsAccordion", disableUnitsAccordion ? "true" : "false")
-  }, [disableUnitsAccordion, setValue])
+    reset({ ...getValues(), disableUnitsAccordion: disableUnitsAccordion ? "true" : "false" })
```

Comment:
IMO this is a weird solution but makes the default value work and does not appear to cause any extra renders

Reply:
I agree and I think a longer term solution (for another issue) should be to refactor FieldGroup, because this isn't the only place we're having to do some weird things because of that field.